doc/SLSKPROTOCOL.md

```diff
@@ -258,7 +256,7 @@ We send this to the server right after the connection has been established. Serv
 00 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36
 30 32 64 34 35 32 39 32 39 01 00 00 00**
 
-### Data Order
+#### Data Order
```

Comment:
Could you revert to the old heading level for these (this one looks too similar to regular bold text)?

Reply:
I've gone for _italics_ instead of a tag, because tagging these sub-headings do not serve any useful purpose for navigation.